crates/glaredb/src/args/mod.rs

```diff
@@ -4,6 +4,8 @@ use clap::{Parser, ValueEnum};
 use std::fmt::Write as _;
 use std::path::PathBuf;
 use url::Url;
+
+use crate::rpc_proxy::TLSMode;
```

Comment:
(I'm a novice Rustacean): I'm wondering if this is needed since it's all in the same crate?

Reply:
Ah, I see, it is needed when in parent dir

Reply:
symbols are file scoped (because translation units still matter apparently), but same.